### ISSUE TITLE
I can't convert to pdf or pptx on Windows Subsystem for Linux

### ISSUE THREAD

**Issue by oren** [marp-team/marp-cli#175]
I am using an Ubuntu shell inside Windows (Windows Subsystem for Linux). Running `--server` and `--html` works but when I try `--pdf` or `--pptx` the command continues forever and I never get the cursor back.

Any idea how to troubleshoot this?

**Comment by yhatt** [comment]
I'll test later in same environment to try reproduce. Some advice I can tell now:

- PDF/PPTX conversion in WSL requires Chrome for Windows: `/mnt/c/Program Files/Google/Chrome/Application/chrome.exe`.  
https://github.com/marp-team/marp-cli/blob/master/src/utils/find-chrome.ts#L11-L12
- The latest Marp CLI is using [Puppeteer v1.20.0](https://github.com/GoogleChrome/puppeteer/releases/tag/v1.20.0) to convert. If you are using outdated Chrome <= 76, it may not return the output because of difference of PDF generation protocol.  
https://github.com/marp-team/marp-cli/pull/106#discussion_r296097319

**Comment by yhatt** [comment]
If you want troubleshoot in myself environment, `DEBUG=*` env would help to check working some dependent modules. (Please careful it might output very huge log)

```
DEBUG=* marp xxxxxx.md
```

**Comment by oren** [comment]
awesome advice. I'll try it out in a few days and will report back

**Comment by yhatt** [comment]
I've confirmed the same issue on Marp for VS Code + Remote WSL. (marp-team/marp-vscode#89)

According to debug log (`DEBUG=*`), [Puppeteer](https://github.com/GoogleChrome/puppeteer) seems to fail execute installed Chrome in Windows host.

```
[  INFO ] Converting 1 markdown...
  puppeteer:protocol SEND ► {"method":"Target.setDiscoverTargets","params":{"discover":true},"id":1} +0ms
  puppeteer:protocol ◀ RECV {"method":"Target.targetCreated","params":{"targetInfo":{"targetId":"170035E506EBF2213D1473583EEC98A7","type":"page","title":"","url":"about:blank","attached":false,"browserContextId":"E9E0AB510BF8FD88BC02217A79F4C278"}}} +4ms
  puppeteer:protocol ◀ RECV {"method":"Target.targetCreated","params":{"targetInfo":{"targetId":"26db2069-c3c7-4a29-9e15-20b5f25cc0b6","type":"browser","title":"","url":"","attached":false}}} +0ms
  puppeteer:protocol ◀ RECV {"method":"Target.targetCreated","params":{"targetInfo":{"targetId":"720114e9-3491-4b37-84a6-ff9968c0057f","type":"browser","title":"","url":"","attached":true}}} +1ms
  puppeteer:protocol ◀ RECV {"id":1,"result":{}} +0ms
  puppeteer:protocol SEND ► {"method":"Target.createTarget","params":{"url":"about:blank"},"id":2} +1ms
  puppeteer:protocol ◀ RECV {"method":"Target.targetCreated","params":{"targetInfo":{"targetId":"5ECF9FB590AA1CDCDCF48B81A23FA7D6","type":"page","title":"","url":"","attached":false,"browserContextId":"E9E0AB510BF8FD88BC02217A79F4C278"}}} +2ms
  puppeteer:protocol ◀ RECV {"id":2,"result":{"targetId":"5ECF9FB590AA1CDCDCF48B81A23FA7D6"}} +1ms
  puppeteer:protocol ◀ RECV {"method":"Target.targetCrashed","params":{"targetId":"5ECF9FB590AA1CDCDCF48B81A23FA7D6","status":"failed to launch","errorCode":2}} +1ms
```

**Comment by yhatt** [comment]
@oren I've confirmed a patched v0.15.1 is working PDF conversion in the WSL environment correctly.

Please report again if you something fails in the latest version of CLI.
